packages/untaped-github/AGENTS.md

```
@@ -51,27 +52,86 @@ AGENTS.md Hard Rule 11). Don't invent your own verify resolution.
 Requests`.
 
+## Search
+
+`search` is a Typer sub-app mounted on the root `github` app, with one
+subcommand per GitHub search endpoint:
+
+| Subcommand            | Endpoint                  | Key filters                                              |
+| --------------------- | ------------------------- | -------------------------------------------------------- |
+| `search repos`        | `/search/repositories`    | `--name`, `--language`, `--archived/--no-archived`, `--fork/--no-fork`, `--visibility`, `--sort` |
+| `search code`         | `/search/code`            | `--language`, `--filename`, `--path`, `--extension`      |
+| `search issues`       | `/search/issues`          | `--state`, `--kind issue|pr`, `--author`, `--assignee`, `--label`, `--mentions` |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)